ast/compile.go

```diff
@@ -104,6 +106,7 @@ type Compiler struct {
 	comprehensionIndices map[*Term]*ComprehensionIndex // comprehension key index
 	initialized          bool                          // indicates if init() has been called
 	schemaSet            *SchemaSet
+	debug                debug.Debug // emits debug information produced during compilation
```

Comment:
Let's update the compiler so that the debug stream is passed to the `compile` function in this file--that way we can run `opa build  --debug x.rego` to quickly determine if comprehensions in that file (or directory, or bundle, or ...) are getting indexed.